backend/src/schemas/slide.schema.ts

```diff
@@ -1,82 +1,8 @@
+import { Schema as MongooseSchema } from 'mongoose';
 import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
 import { Board } from './board.schema';
+import { SlideObject } from './slide-object.schema';
 
```

Comment:
```typescript
export type SlideDocument = Slide & MongooseDocument
```

Reply:
Not really in scope of this PR, but done.